### PR TITLE
UI text consistency, capitalization etc.

### PR DESCRIPTION
**Pull request by torhus**
I've probably forgotten something, please let me know.

### CODE REVIEW

File: src/fe-gtk/setup.c [hunk 1]
```diff
@@ -556,14 +556,14 @@ static const setting advanced_settings[] =
 	{ST_HEADER,	N_("Auto Copy Behavior"),0,0,0},
 	{ST_TOGGLE, N_("Automatically copy selected text"), P_OFFINTNL(hex_text_autocopy_text),
 					N_("Copy selected text to clipboard when left mouse button is released. "
-						"Otherwise, CONTROL-SHIFT-C will copy the "
+						"Otherwise, Ctrl+Shift+C will copy the "
```
Comment (TingPing) [inline src/fe-gtk/setup.c]: I think we should use `Control`.

Comment (torhus) [inline src/fe-gtk/setup.c]: Ok, any particular reason? I just followed this: https://developer.gnome.org/hig/stable/keyboard-input.html.en

Comment (TingPing) [inline src/fe-gtk/setup.c]: Hmm, nevermind you are right and it matches our keyboard shortcuts dialog.

Comment (torhus) [inline src/fe-gtk/setup.c]: Cool :)

**Pull request merged**